index.js

```diff
@@ -1,28 +1,34 @@
+/* eslint-disable no-multiple-empty-lines */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-empty */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-multi-spaces */
+/* eslint-disable quotes */
 /* eslint-disable prefer-destructuring */
```

Comment:
- [ ] As suggested by the reviewer before please remove the inline comments to disable linters. Linters are there to ensure best practices are used and  all the required rules are already in your provided [linter file](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) by microverse.